ui/package.json

```diff
@@ -20,7 +20,7 @@
     "@atlaskit/theme": "^12.1.6",
     "@atlaskit/tokens": "^1.29.1",
     "@atlaskit/tooltip": "^17.5.9",
-    "@atlassian/forge-graphql": "13.3.10",
+    "@atlassian/forge-graphql": "13.3.13",
```

Comment:
Seeing the build errors, @VladyslavLevinUA, this is going to require running `yarn install` and committing the changes as part of the PR, once we verify that every related dependency bump is still okay.